src/contexts/ThemeContext.tsx

```diff
@@ -27,7 +27,7 @@ export const ThemeContext = createContext<IThemeContext>({
 export const ThemeProvider: React.FC<IProps> = ({ children }: IProps) => {
   const [theme, setNewTheme] = useState(initialTheme);
   const setTheme = useCallback((name: string) => {
-    const newTheme = _setTheme(name, initialTheme.name);
+    const newTheme = _setTheme('magic', initialTheme.name);
```

Comment:
This is not DRY (Don't Repeat Yourself). What you did in this file is you changed the theme in a couple different places. Why not just make it so there's only 1 place? Just change line `35` so you always return the default theme (`theme: defaultThemeName`). The rest of the changes does not matter. Please also add a comment above that line to say something like `// force the default theme` so if we come back to this some time later then it all makes sense (it's one of those rare cases when a comment can actually be useful)